src/gui/tray/UserModel.cpp

```diff
@@ -58,52 +59,54 @@ User::User(AccountStatePtr &account, const bool &isCurrent, QObject *parent)
     connect(_activityModel, &ActivityListModel::sendNotificationRequest, this, &User::slotSendNotificationRequest);
 }
 
-void User::slotBuildNotificationDisplay(const ActivityList &list)
+void User::showOsNotification(const QString &title, const QString &message)
```

Comment:
s/Os/Desktop/ ?

These two letter acronyms stuck between other words are always harder to spot. :-)